pkg/epp/handlers/response.go

```diff
@@ -61,16 +61,26 @@ func (s *StreamingServer) HandleResponseBody(ctx context.Context, reqCtx *Reques
 	reqCtx.ResponseComplete = true
 
 	reqCtx.respBodyResp = generateResponseBodyResponses(responseBytes, true)
+	if s.director != nil {
+		s.director.HandleResponseBodyComplete(ctx, reqCtx)
+	}
```

Comment:
don't need to check director != nil on every call. 
this is verified on StreamingServer initialization.
```suggestion
	s.director.HandleResponseBodyComplete(ctx, reqCtx)
```